packages/native/test/native-css.test.js

```diff
@@ -0,0 +1,115 @@
+import { css } from '@emotion/native'
+
+let returnArguments = (...args) => args
+
+describe('Emotion native css', () => {
```

Comment:
I'm not really sure if we need all these tests duplicated, maybe it could just be a small smoke test to make sure css works

Reply:
I was thinking of this too! I'll need some help over here so will ping you later when I comeback to this after fixing basic stuff.